src/components/timeline/ActiveTabGlobalTrack.js

```diff
@@ -135,6 +158,7 @@ export default explicitConnect<OwnProps, StateProps, DispatchProps>({
 
     // These get assigned based on the track type.
     let isSelected = false;
+    let resourceTracks = EMPTY_RESOURCE_TRACKS;
```

Comment:
The following is non-blocking for this PR, but something to think about:

Why isn't the `getActiveTabResourceTracks` selector just run here? I feel like there are some assumptions on how this selector run that aren't encoded into the code and type system.

From what I'm reading there should only be one main tab track, but then can be mulltiple other types of global tab track. The `getActiveTabResourceTracks` will only return one one list, and that needs to be displayed only once. This feels like the implementation will _work_ as it's written, but could break if we ever had more than one `tab` type track, or 0 `tab track`.

I don't think we need to change the implementation now, but it may be something to think about.

Reply:
Yeah, we should think about it. Currently there can be multiple main tracks because we don't merge the tracks. But we can solve it by merging the tracks if there are multiple main tracks in the future. Having 0 main track should not be possible. I will add a check in the receive-profile path, and if it's 0, I will redirect this to full view instead.